src/Ledger/Address.lagda

```diff
@@ -22,7 +23,8 @@ credential contains a hash, either of a verifying (public) key
   ScriptHash
 \end{code}
 \begin{code}[hide]
-  : Type) ⦃ _ : DecEq Network ⦄ ⦃ _ : DecEq KeyHash ⦄ ⦃ _ : DecEq ScriptHash ⦄ where
+  : Type)  ⦃ _ : DecEq Network ⦄ ⦃ _ : DecEq KeyHash ⦄ ⦃ _ : DecEq ScriptHash ⦄
+           ⦃ _ : Show Network  ⦄ ⦃ _ : Show KeyHash  ⦄ ⦃ _ : Show ScriptHash  ⦄ where
```

Comment:
I wouldn't add the `Show` instances as module parameters since they're only needed for defining other `Show` instances. It'd be better to move these parameters to the `Show` instance definitions that are actually using them.

Reply:
Yeah. Might be worth considering doing the same for `DecEq`, but there might be some stuff that requires it that isn't some other `DecEq` instance in which case I think this might be cleaner.